kb/2022-04-06/nic_naming_scheme.md

```diff
@@ -0,0 +1,78 @@
+---
+title: NIC Naming Scheme
+descripion: NIC Naming Scheme Change
+slug: nic-naming-scheme
+authors:
+  - name: Date Huang
+    title: Software Engineer
+    url: https://github.com/tjjh89017
+    image_url: https://github.com/tjjh89017.png
+tags: [network]
+hide_table_of_contents: false
+---
+
+# NIC Naming Scheme
+
+## NIC Naming Scheme is changed after upgrade v1.0.1
+
+`systemd` in OpenSUSE Leap 15.3 which is the base OS of Harvester is upgraded to `246.16-150300.7.39.1`. In this version, `systemd` will enable additional nameing scheme `sle15-sp3` which is `v238` with `bridge_no_slot`. When there is a PCI bridge assocated with NIC, `systemd` will never generate `ID_NET_NAME_SLOT` and naming policy in `/usr/lib/systemd/network/99-default.link` will fallback to `ID_NET_NAME_PATH`. According to this change, NIC names might be changed in your Harvester nodes during the upgrade process from `v1.0.0` to `v1.0.1-rc1` or above and it will cause network issues which are assocated with NIC names.
```

Comment:
```suggestion
`systemd` in OpenSUSE Leap 15.3 which is the base OS of Harvester is upgraded to `246.16-150300.7.39.1`. In this version, `systemd` will enable additional naming scheme `sle15-sp3` which is `v238` with `bridge_no_slot`. When there is a PCI bridge associated with NIC, `systemd` will never generate `ID_NET_NAME_SLOT` and naming policy in `/usr/lib/systemd/network/99-default.link` will fallback to `ID_NET_NAME_PATH`. According to this change, NIC names might be changed in your Harvester nodes during the upgrade process from `v1.0.0` to `v1.0.1-rc1` or above, and it will cause network issues that are associated with NIC names.
```